### DISCUSSION TITLE
PRO badge not appearing

### DISCUSSION THREAD

**Discussion by saisumith770** [community/community#69634] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've enrolled in the student program but cannot see the PRO badge on my github profile! Is there some sort of a waiting period

**Comment by radar-blip** [comment]
There may be a checkbox in your settings that needs to be checked.
Similar (and right above) the Show Achievements checkbox.

![Screenshot from 2023-10-10 00-12-49](https://github.com/community/community/assets/39196767/d0ba079d-f6be-4e6a-b823-56bac785f0b9)

**Comment by santoshvandari** [comment, marked as answer]
Hi "saisumith770,"

Congratulations on enrolling in the student program on GitHub. Usually, the PRO badge should appear on your GitHub profile relatively soon after you've been approved for the GitHub Student Developer Pack. However, there could be a few reasons why it's not showing up immediately.

Here are some things to consider:

Verification Process: Double-check to ensure that GitHub has successfully verified your enrollment in the student program. Before the PRO badge appears on your profile, there might be a brief processing period.

Profile Refresh: Sometimes, the badge may not appear immediately due to browser caching or other technical issues. Try refreshing your GitHub profile page, logging out and back in, or clearing your browser cache to see if the badge appears.

GitHub Support: If you've waited for a reasonable amount of time and the badge still isn't showing up, or if you encounter any other issues, consider reaching out to [GitHub Support](https://support.github.com/contact). They can investigate and assist. You can contact GitHub Support here: GitHub Support.

Check Your Eligibility: Ensure that you meet all the eligibility criteria for the GitHub Student Developer Pack. This includes being a student and having an eligible academic email address. If there are any discrepancies in your eligibility, it might affect the badge display.

Patience: Sometimes, badge updates may be delayed due to high demand or technical reasons. Give it a little more time; it should appear eventually if you've been approved for the Student Developer Pack.

In most cases, the PRO badge should appear on your profile relatively quickly once approved. If it doesn't appear after a reasonable waiting period or you have other concerns, don't hesitate to contact GitHub Support for assistance.

I hope this helps resolve the issue and you can enjoy the benefits of the GitHub Student Developer Pack soon.
